_component_design/toggle.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Toggle
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: >
+  A toggle controls how information is formatted on a control panel page.
+preview-image: preview-images/toggles.svg
+status: stable
+resource: true
+last-modified: 2019-03-11
+helix-ui-css: true
+helix-ui-javascript: true
+pagelink: https://rackerlabs.github.io/helix-ui/components/toogle/
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use a toggle when you want to provide the user control over how information is presented on a control panel page.
+
+For example, the user can use a toggle to switch the display of information from a card view, which displays information on a set of tiles, to a list view.
```

Comment:
...from a card view, which displays each entry as a tile on a grid, to a list view, which displays each entry as an item in a list.

Reply:
Done.